tests/metrics/test-jc.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+# SPDX-FileCopyrightText: Copyright (c) 2021-2025 Yegor Bugayenko
+# SPDX-License-Identifier: MIT
+
+set -e
+set -o pipefail
+
+temp=$1
+stdout=$2
+
+script_location="${LOCAL}/metrics/jc.py"
+
+{
+    java_file="${temp}/TestDoc.java"
+    cat > "${java_file}" <<EOT
+public class TestDoc {
+    /**
+     * This is a documented method.
+     */
+    public void documented() {}
+
+    public void notDocumented() {}
+
+    /**
+     * Another documented method.
+     */
+    public void alsoDocumented() {}
+}
+EOT
+
+    metrics_file="${temp}/metrics.txt"
+    "${script_location}" "${java_file}" "${metrics_file}"
+    cat "${metrics_file}"
+
```

Comment:
@veledara what's the point of double empty line here? Read this: https://www.yegor256.com/2014/11/03/empty-line-code-smell.html